CairoMakie/src/primitives.jl

```diff
@@ -458,34 +458,6 @@ end
 
 regularly_spaced_array_to_range(arr::AbstractRange) = arr
 
-"""
-    interpolation_flag(is_vector, interp, wpx, hpx, w, h)
-
-* is_vector: if we're using vector backend
-* interp: does the user want to interpolate?
-* wpx, hpx: projected size of the image in pixels, so the actual width in pixels on screen
-* w, h: size of image in pixels
-"""
-function interpolation_flag(is_vector, interp, wpx, hpx, w, h)
```

Comment:
Why not leave this in a function?

Reply:
Because the flag logic didn't work anyway, there's no point in switching between the three, there's only two basically. So the main logic isn't about picking the flag but deciding whether to take the fast path or not.